### PR TITLE
[Ready for Review] Conflict resolution selector

### PR DESCRIPTION
**Pull request by n1k0**
/cc @glasserc

### CODE REVIEW

**Comment by n1k0** [comment]
I think this is now ready for review. r=? @glasserc

File: scripts/components/CollectionList.js [hunk 1]
```diff
@@ -123,13 +145,16 @@ class Table extends Component {
...
-              record={record}
-              schema={schema}
-              config={config}
-              deleteRecord={deleteRecord}
-              updatePath={updatePath} />;
+            return (
+              <Row key={index}
+                name={name}
+                record={record}
+                conflict={conflicts[record.id]}
```
Comment (glasserc) [inline scripts/components/CollectionList.js]: Wait, how are conflicts returned? I assumed it was just a list, but it looks more like a Map or something here.

Comment (glasserc) [inline scripts/components/CollectionList.js]: Ah, I see, it changes in this commit.

**Comment by glasserc** [comment]
r+

**Pull request merged**